controller/index.js

```diff
@@ -2,7 +2,7 @@ const scaffold = require('../lib/scaffold');
 const package = require('../lib/package');
 
 const userDependencies = {
-  "password": "git+https://github.com/Prismatik/password.git"
+  "simple-password": "^1.0.1"
```

Comment:
I think we can remove this and put it back in `base/package.mustache` yeah?  I think this `userDependencies` only existed because it was a `git+https` link?